app/templates/passwords/reset-password.html

```diff
@@ -50,24 +55,26 @@ <h1 class="saturn">Reset your password</h1>
     </ul>
 
     <fieldset id="reset-details" class="js-new-password-group">
-        {% if errorType != None %}
+
+        {% if errorType.password %}
```

Comment:
The password field is displaying an error with javascript code when the password complexity is not met. There is an identical problem on the register-account-details page so could fix there too